hasher-matcher-actioner/webapp/src/components/settings/ActionRulesTableRow.jsx

```diff
@@ -62,6 +81,48 @@ export default function ActionRulesTableRow({
     return <span>&mdash;</span>;
   };
 
+  const getClassificationDescriptions = () => {
+    const classificationDescriptions = updatedActionRule.classifications.map(
+      classification => {
+        let ret = 'the';
+        switch (classification.classification_type) {
+          case 'BankSourceClassification':
+            ret += ' Dataset Source';
+            break;
+          case 'BankIDClassification':
+            ret += ' Dataset ID';
+            break;
+
+          case 'BankedContentIDClassification':
+            ret += ' MatchedContent ID';
+            break;
+
+          case 'Classification':
```

Comment:
The Classification case seems a little awkward to me. seems like at line 115 and 117 below you could have inline conditional. For example line 115 could be 

`ret += classification.classification_type === 'Classification' ? ' has been classified as ' : ' is ';`

And if you do that (put the trailing spaces in ret at this point, then you can drop the back quote ${...} on line 119.

Reply:
Anyways, sorry if too in the weeds, but the fact that it returns inside the switch in one case and falls through in another rubs me the wrong way for whatever reason.

Reply:
Agreed. Does this new logic seem cleaner (reproduced):

```
let ret = 'the';
        switch (classification.classification_type) {
          case 'BankSourceClassification':
            ret += ' Dataset Source';
            break;
          case 'BankIDClassification':
            ret += ' Dataset ID';
            break;

          case 'BankedContentIDClassification':
            ret += ' MatchedContent ID';
            break;

          case 'Classification':
            ret += ' MatchedContent';
            break;
          default:
            ret += ` ${classification.classification_type}`;
            break;
        }

        if (classification.equals) {
          ret +=
            classification.classification_type === 'Classification'
              ? ' has been classified'
              : ' is';
        } else {
          ret +=
            classification.classification_type === 'Classification'
              ? ' has not been classified'
              : ' is not';
        }
        ret += ` ${classification.classification_value}`;
        return ret;
        ```